File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_20635.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest20635():
    multipart_value = request.form.get('multipart_field', '')
    def normalize(value):
        return value.strip()
    data = normalize(multipart_value)
    return jsonify({'error': 'An internal error occurred'}), 500

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest20635.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest20635 {

    @PostMapping(path="/BenchmarkTest20635", consumes="application/xml")
    public void BenchmarkTest20635(@RequestBody String xmlBody, HttpServletRequest request, HttpServletResponse response) throws Exception {
        response.sendError(500, "Internal error");
    }
}
